pdg/src/info.rs

```diff
@@ -0,0 +1,25 @@
+use crate::Graphs;
+use std::fmt::{self, Debug, Display, Formatter};
+
+/// Information generated from the PDG proper that is queried by static analysis.
+///
+/// Eventually this will include information about what kinds of [`Node`]s the [`Node`] flows to,
+/// as well as its ability to be used as a `&mut`.
+///
+/// [`Node`]: crate::graph::Node
+#[derive(Hash, Clone, PartialEq, Debug)]
+pub struct NodeInfo {}
+
+impl Display for NodeInfo {
+    fn fmt(&self, f: &mut Formatter) -> fmt::Result {
+        write!(f, "")
+    }
+}
+
+pub fn add_info(pdg: &mut Graphs) {
```

Comment:
nitpick: `add_empty_info` or `set_default_info`? to me `add_info` implies that there is actually going to be something set in `NodeInfo`

Reply:
The very next change is going to set NodeInfo to something real. Should this have a more descriptive name after this one PR and then immediately change it back?

Reply:
I would change the name to what you anticipate it becoming, perhaps utilizing `todo!(...)`, and adding documentation for what the function will do

Reply:
I disagree.  I think documentation would help (saying what it's meant to do), but I wouldn't change the name.  This sets `NodeInfo` as is, which is enough right now.  Once `NodeInfo` has fields, it'll set something.  `todo!` will also crash, and this function is called.

Reply:
> todo! will also crash, and this function is called.

fair enough. as for the rest, i don't really feel strongly but that's what i'd do

Reply:
I think it doesn't matter too much as it'll be changed soon enough.  @boyland-pf, just add some docs and I think it'll be good, and then you can open the next PR.